core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -81,12 +83,14 @@ public class JdbcConfig {
   public JdbcConfig(DatabaseConfig databaseConfig) {
     String storage = databaseConfig.getStorage();
     String transactionManager = databaseConfig.getTransactionManager();
-    if (!"jdbc".equals(storage) && !"jdbc".equals(transactionManager)) {
+    if (!storage.equals(STORAGE_NAME) && !transactionManager.equals(STORAGE_NAME)) {
```

Comment:
Currently, when we set the `transactionManager` config to `jdbc`, the `storage` config is not required. So the `storage` config can be null. So we need to use the `STORAGE_NAME.equals(...)` style to avoid `NullPointerExceptiom`:
```suggestion
    if (!STORAGE_NAME.equals(storage) && !STORAGE_NAME.equals(transactionManager)) {
```

Or we also use `Objects.equals()` as well. Either is fine for me. Thanks.

Reply:
@brfrn169 Thank you for pointing it out. I think `cassandra` is always set to the storage by default when we create `DatabaseConfig`, it will never be null. If I understand it correctly, is the assertion better?

Reply:
Ah yes, you are right. I don't remember why we used that reverse style (`"jdbc".equals(storage)`) originally. It looks like we have default values for `storage` and `transactionManager`, so we don't need to care about `NullPointerException`. So let's keep it as it is. Thanks.